hl7-mllp/2.1/modules/ROOT/edits/hl7-mllp-connector-examples.adoc

```diff
@@ -0,0 +1,136 @@
+= HL7 EDI - Examples - Mule 4
+
+Use these examples to read and write HL7 messages over the MLLP protocol.
+
+== Example: HL7 Studio
+
+Set these Global properties before using the examples:
+
+* Delimiter: The standard HL7 delimiter value that separates
+MLLP messages. It is a hexadecimal 0x1C character also known as FS. Applications in some clinics use a different delimiter value, that can be set in this field. The standard value in the
+HL7 MLLP connector is shown as `001C`.
++
+* Host: Sets the socket host address.
+* Port: Sets the socket port.
++
+For an advanced connection, complete these fields:
++
+* Reuse address: Whether to reuse the configured socket or fail when trying to bind to the socket.
+* Send TCP With No Delay: Whether or not to transmit data immediately, or collect data for greater efficiency before sending.
+* Linger: How long in milliseconds that the socket takes to close so any remaining data transmits correctly.
+* Keep alive: Whether to close open socket connections that are unused for a long period and with an unavailable connection.
+* Fail on unresolved host: Whether the socket should fail during its creation if the host set on the endpoint cannot be resolved.
+
+* In the TLS tab you can configure a new TLS configuration, its presence implies the use of SSLServerSocket instead of plain TCP.
+
+* Set the send or receive buffer size.
+
+== Example - Read and Write Message
+
+image::hl7-mllp-demo-flow-1.png[MLLP Demo Flow in Studio 7]
+
+image:hl7-mllp-demo-flow-2.png[MLLP Demo Flow in Studio 7]
+
+image::hl7-mllp-demo-flow-3.png[MLLP Demo Flow in Studio 7]
+
+This example workflow sequentially invokes read and write HL7 messages over the MLLP protocol:
+
+* writer-flow: Sends the given message through the socket to be received by the reader-flow.
+
+* reader-flow: Receive a MLLP message, and log the message.
```

Comment:
Reformat as proper bullet list items and remove periods; make parallel:

```suggestion
* writer-flow
+
Sends the given message through the socket to be received by the reader flow

* reader-flow
+
Receives and logs an MLLP message
```

Reply:
fixed.